sys-devel/llvm/patches/clang-6.0.1.patchset

```diff
@@ -170,5 +170,37 @@ index 8b5b48e..ccd851f 100644
    std::string findLibCxxIncludePath() const override;
    void addLibStdCxxIncludePaths(
 -- 
-2.16.4
+2.19.1
+
+
+From d19f9f1dbd2151f68d8810f2820f6b62f596d9d2 Mon Sep 17 00:00:00 2001
+From: Calvin Hill <calvin@hakobaito.co.uk>
+Date: Tue, 16 Oct 2018 17:44:52 +0100
+Subject: clang: Enable float128 support in clang
+
+
+diff --git a/lib/Basic/Targets/OSTargets.h b/lib/Basic/Targets/OSTargets.h
+index 790732b..3ff1fbc 100644
+--- a/lib/Basic/Targets/OSTargets.h
++++ b/lib/Basic/Targets/OSTargets.h
+@@ -251,6 +251,9 @@ protected:
+     Builder.defineMacro("__HAIKU__");
+     Builder.defineMacro("__ELF__");
+     DefineStd(Builder, "unix", Opts);
++    if (this->HasFloat128)
```

Comment:
Missing braces?

Reply:
That was done intentionally to be consistent with LLVM's code style guidelines. A similar [patch](https://reviews.llvm.org/D41240#change-onZRG3.R04Cv) for many targets like OpenBSD, Solaris, etc leave out the braces in OSTargets.h. So I think this should be acceptable even for upstreaming.

Reply:
Except you have 2 lines indented as if they are part of the if, but only 1 is actually encapsulated by the if.

Reply:
The patch has now been updated.